editaveis/solucaoeletronica.tex

```diff
@@ -0,0 +1,367 @@
+
+\chapter{Eletrônica}
+
+Para solucionar o problema da implementação de um radar de efeito doppler, foi estipulado uma solução dividida em vários sistemas integrados. Onde todos esses sistemas estariam dentro de um totem dispostos de acordo com a figura \ref{esquematico1}. Nesta disposição os sistemas estariam divididos em funcionamento do sinal de radar, sistema de processamento de sinais, sistema de controle e sistema de comunicação.
+
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale=0.85]{totem.pdf}
+    \caption{Esquemático da solução eletrônica proposta para a construção do totem. }
+    \label{esquematico1}
+\end{figure}
+
+\section{Sistema do radar}
+%Descrever o por que da configuração monoestática, possivelmente custo de projeto.
+
+O radar será implementado na configuração monoestática, isto é, uma antena realiza transmissão e recepção do sinal. Devido a este fator, a escolha da forma de funcionamento do radar, o pulso será modulado em largura através da técnica de modulação \emph{Pulse Width Modulation} (PWM). Portanto, uma fração do período de repetição dos pulsos corresponderão a uma janela de tempo para a \emph{Field Programable Array} (FPGA) processar a informação recebida da antena, e a outra parte para enviar o sinal. Esse tipo de configuração foi escolhida pela diminuição dos custos do projeto.
+Nas próximas seções referentes ao radar, estarão detalhados os blocos do diagrama da Figura \ref{processos_geral_radar}, onde cada bloco será expandido para explicar seu funcionamento.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale=1]{sistema_radar.pdf}
+    \caption{Diagrama geral do radar representando a responsabilidade de cada bloco na detecção do veículo e estimativa da velocidade.}
+    \label{processos_geral_radar}
+\end{figure}
+
+\subsection{Parâmetros do radar}
+%Da pra detalhar melhor esse parágrafo
+O projeto consiste em transmitir e receber um sinal para detectar a aproximação de um veículo, e após detectar, analisar o desvio de frequência para estimar sua velocidade. Para realização deste processo, é preciso definir alguns parâmetros.
+
+O primeiro parâmetro é a distância máxima que o radar vai conseguir captar um veículo. A partir disso foi estipulado uma distância de 100m, visto que com essa distância é possível alertar o motorista que vem em sentido contrário, além de dar tempo para que seja possível capturar uma imagem se o veículo estiver acima da velocidade da via. A partir disso foi possível realizar o cálculo dos demais parâmetros.
+
+Após isso foi calculado a frequência de repetição de pulso (PRF), que é o número de ciclos de transmissão e é calculado por:
+\begin{equation}\label{R_MAX}
+    R_{max} =  \frac{c}{2PRF},
+\end{equation}
+onde c é a velocidade da luz e $R_{max}$ é a distância máxima de alcance do radar. Com isso temos que a PRF é de,
```

Comment:
acredito que esse "c" da velocidade da luz tem que estar em itálico, pois se refere a uma variável da fórmula